ui/app/AppLayouts/Chat/panels/communities/HoldingsListPanel.qml

```diff
@@ -0,0 +1,95 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+
+import AppLayouts.Chat.helpers 1.0
+import AppLayouts.Chat.controls.community 1.0
+
+import SortFilterProxyModel 0.2
+
+import utils 1.0
+
+Control {
+    id: root
+
+    property var model
+    property string introText
+
+    QtObject {
+        id: d
+
+        // By design values:
+        readonly property int defaultHoldingsSpacing: 8
+
+        function holdingsTextFormat(name, amount) {
+            return CommunityPermissionsHelpers.setHoldingsTextFormat(HoldingTypes.Type.Asset, name, amount)
+        }
+    }
+
+    contentItem: ColumnLayout {
+        spacing: root.spacing
+
+        StatusBaseText {
+            Layout.alignment: Qt.AlignHCenter
+            text: root.introText
+            textFormat: Text.StyledText
+        }
+
+        ColumnLayout {
+            Layout.alignment: Qt.AlignHCenter
+            spacing: d.defaultHoldingsSpacing
+
+            Repeater {
+                id: repeater
+                model: root.model
+
+                ColumnLayout {
+                    Layout.alignment: Qt.AlignHCenter
+                    spacing: d.defaultHoldingsSpacing
+
+                    RowLayout {
+                        spacing: 18 // by design
+
+                        Repeater {
+                            model: SortFilterProxyModel {
+                                sourceModel: holdingsListModel
```

Comment:
what is `holdingsListModel` in this context?

Reply:
`HoldingsListPanel` has a model that represents a permission.
A permission consists on a list of different properties, one of them is `holdings list`. At this point, the repeater is working with / representing this `holdings` list model.